review04/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/repository/PassengerRepositoryTest.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.airportmanager.passengerservice.repository;
+
+import cz.muni.fi.airportmanager.passengerservice.entity.Notification;
+import cz.muni.fi.airportmanager.passengerservice.entity.Passenger;
+import cz.muni.fi.airportmanager.passengerservice.model.NotificationDto;
+import io.quarkus.test.TestReactiveTransaction;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.vertx.UniAsserter;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+class PassengerRepositoryTest {
+
+    @Inject
+    PassengerRepository passengerRepository;
+
+
+    @Test
+    @TestReactiveTransaction
+    void shouldFindNotificationsForPassenger(UniAsserter asserter) {
+        Passenger passenger = createTestPassenger();
+        Notification notification = createTestNotification();
+
+        asserter.execute(() -> passengerRepository.deleteAll())
```

Comment:
I'm also not sure if you need to deletetAll every time if you have TestReactiveTransaction

Reply:
removed